rest/sgcollect.go

```diff
@@ -90,69 +95,46 @@ func (sg *sgCollect) Start(logFilePath string, ctxSerialNumber uint64, zipFilena
 
 	zipPath := filepath.Join(params.OutputDirectory, zipFilename)
 
-	args := params.Args()
-	args = append(args, "--sync-gateway-executable", sgPath)
-	args = append(args, zipPath)
+	cmdline := slices.Clone(sg.sgCollectPath)
+	cmdline = append(cmdline, params.Args()...)
+	cmdline = append(cmdline, "--sync-gateway-executable", sg.sgPath)
+	cmdline = append(cmdline, zipPath)
 
-	ctx := base.CorrelationIDLogCtx(context.Background(), fmt.Sprintf("SGCollect-%03d", ctxSerialNumber))
+	ctx, sg.cancel = context.WithCancel(ctx)
+	cmd := exec.CommandContext(ctx, cmdline[0], cmdline[1:]...)
 
-	sg.context, sg.cancel = context.WithCancel(ctx)
-	cmd := exec.CommandContext(sg.context, sgCollectPath, args...)
-
-	// Send command stderr/stdout to pipes
-	stderrPipeReader, stderrPipeWriter := io.Pipe()
-	cmd.Stderr = stderrPipeWriter
-	stdoutPipeReader, stdoutpipeWriter := io.Pipe()
-	cmd.Stdout = stdoutpipeWriter
+	outStream := newSGCollectOutputStream(ctx, sg.stdout, sg.stderr)
+	cmd.Stdout = outStream.stdoutPipeWriter
+	cmd.Stderr = outStream.stderrPipeWriter
 
 	if err := cmd.Start(); err != nil {
+		outStream.Close(ctx)
 		return err
 	}
 
 	atomic.StoreUint32(sg.status, sgRunning)
 	startTime := time.Now()
-	base.InfofCtx(sg.context, base.KeyAdmin, "sgcollect_info started with args: %v", base.UD(args))
-
-	// Stream sgcollect_info stderr to warn logs
-	go func() {
-		scanner := bufio.NewScanner(stderrPipeReader)
-		for scanner.Scan() {
-			base.InfofCtx(sg.context, base.KeyAll, "sgcollect_info: %v", scanner.Text())
-		}
-		if err := scanner.Err(); err != nil {
-			base.ErrorfCtx(sg.context, "sgcollect_info: unexpected error: %v", err)
-		}
-	}()
-
-	// Stream sgcollect_info stdout to debug logs
-	go func() {
-		scanner := bufio.NewScanner(stdoutPipeReader)
-		for scanner.Scan() {
-			base.InfofCtx(sg.context, base.KeyAll, "sgcollect_info: %v", scanner.Text())
-		}
-		if err := scanner.Err(); err != nil {
-			base.ErrorfCtx(sg.context, "sgcollect_info: unexpected error: %v", err)
-		}
-	}()
+	base.InfofCtx(ctx, base.KeyAdmin, "sgcollect_info started with cmdline: %v", base.UD(cmdline))
```

Comment:
I know this was how it was before this PR, but let's remove the args for additional clarity

```
2025-06-24T14:28:26.672+01:00 [INF] HTTP: c:#001 POST /_sgcollect_info (as <ud>Administrator</ud> as ADMIN)
2025-06-24T14:28:26.673+01:00 [DBG] Admin+: c:SGCollect-001 sgcollect_info: no output directory specified, using LogFilePath: /tmp/sglog-resync-test
2025-06-24T14:28:26.677+01:00 [INF] Admin: c:SGCollect-001 sgcollect_info started with cmdline: [ <ud>/Users/benbrooks/dev/cb/sg/tools/sgcollect_info</ud> <ud>--sync-gateway-username</ud> <ud>Administrator</ud> <ud>--sync-gateway-password</ud> <ud>password</ud> <ud>--sync-gateway-url</ud> <ud>http://127.0.0.1:4985</ud> <ud>--sync-gateway-executable</ud> <ud>/Users/benbrooks/dev/cb/sg/sync_gateway</ud> <ud>/tmp/sglog-resync-test/sgcollectinfo-2025-06-24t132826-sg@192.168.68.210.zip</ud> ]
```

It's still worth printing out `zipPath` since that's a generated name and users would probably want to know where it will be

```suggestion
	base.InfofCtx(ctx, base.KeyAdmin, "sgcollect_info started with output zip: %v", base.UD(zipPath))
```